### PR TITLE
Use meilisearch from mini-lab #34

### PR DESCRIPTION
**Pull request by vknabel**
This change is split over the following PRs to implement https://github.com/metal-stack-cloud/deployment/issues/34:

- https://github.com/metal-stack-cloud/ansible-roles/pull/11
- https://github.com/metal-stack-cloud/deployment/pull/35

It removes the meilisearch ansible role from [metal-stack-cloud/ansible-roles](https://github.com/metal-stack-cloud/ansible-roles) and replaces it with the already existing [metal-stack/mini-lab](https://github.com/metal-stack/mini-lab).

metal-stack will use the `metal-` prefix, while the api-server of metal-stack-cloud uses the `cloud-` prefix in meilisearch.

**How to test?**

- checkout deployment `feature/use-metal-auditing` [metal-stack-cloud/deployment#35](https://github.com/metal-stack-cloud/deployment/pull/35)
- either link the ansible-roles in `deployment/docker-compose.yml` at branch `feature/use-metal-auditing`...
- ...or in `deployment/requirements.yaml` change the version from `main` to `feature/use-metal-auditing`
- `make up`
- visit http://auditing.172.17.0.1.nip.io:8080/
- enter the secret master key `geheim` (only for debugging!)
- you should see at least two indexes
- one prefixed with `metal-`
- one prefixed with `cloud-`
- the more you do in the console or by using the metalctl, the more entries you should see

image

### CODE REVIEW

File: roles/api-server/defaults/main/main.yaml [hunk 1]
```diff
@@ -27,10 +27,11 @@ api_server_masterdata_api_port: 8443
 api_server_metal_api_url: http://metal-api.metal-control-plane.svc.cluster.local:8080
 api_server_metal_api_hmac: metal-admin
 
-api_server_auditing_url: http://meili-meilisearch.{{ metal_stack_cloud_namespace }}.svc.cluster.local:7700
+api_server_auditing_url: http://auditing-meili-meilisearch.metal-control-plane.svc.cluster.local:7700
```
Comment (majst01) [inline roles/api-server/defaults/main/main.yaml:30]: maybe now we need to create a `auditing` namespace and put meilisearch there ? WDYT ?

Comment (vknabel) [reply roles/api-server/defaults/main/main.yaml:30]: Good point. Introducing an `auditing` namespace would probably be cleaner as we now know that we might introduce alternative backends in the future. 

But I am not sure if it's worth the effort _at the moment_ to extract auditing into a new role in the mini-lab.

Comment (Gerrit91) [reply roles/api-server/defaults/main/main.yaml:30]: We can keep this in the mini-lab as it is, but extend the meilisearch role to have a configurable namespace. For other environments we can then split up and deploy meilisearch into a different namespace.

Comment (vknabel) [reply roles/api-server/defaults/main/main.yaml]: And merged. :)

**Review by majst01 — APPROVED** [review]

**Pull request merged**